packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.stories.ts

```diff
@@ -52,6 +52,10 @@ export const Default = {
       h(OnyxButton, { label: "Slot content goes here", ...(trigger as any) }),
     icon: circleInformation,
     open: true,
+    position: "auto",
+    color: "neutral",
+    float: "auto",
+    de,
```

Comment:
"de" is throwing an error in Storybook, it's not needed here :)

![image](https://github.com/user-attachments/assets/304b226d-6303-4c29-8742-99777d5c3901)